src/expressions/antlr-interface/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ Set(SRCS
 
 add_library(${PROJ} ${SRCS})
 add_library(Antares::${PROJ} ALIAS ${PROJ})
+if (NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
What if we compile on Windows ? What's happening then ?

Reply:
Nothing